lib/Kconfig.debug

```diff
@@ -34,6 +34,10 @@ config PRINTK_CALLER
 	  Since toggling after boot makes the code racy, currently there is
 	  no option to enable/disable at the kernel command line parameter or
 	  sysfs interface.
+	  
```

Comment:
Please remove whitespaces from lines 37 and 38, kconfig is complaining: "lib/Kconfig.debug:40:warning: leading whitespace ignored"